apiserver/errors/types.go

```diff
@@ -124,10 +149,10 @@ func (e *NotLeaderError) ServerAddress() string {
 }
 
 // Is returns true if the target is the NotLeaderError.
-//func (e *NotLeaderError) Is(target error) bool {
+// func (e *NotLeaderError) Is(target error) bool {
```

Comment:
Maybe we should just delete this function, since despite the whitespace fix here, it appears, based on being commented out.. unused.